overlord/snapstate/snapstate.go

```diff
@@ -427,6 +428,41 @@ func CheckChangeConflict(st *state.State, snapName string, checkConflictPredicat
 	return nil
 }
 
+func contentAttr(attrs map[string]interface{}) string {
+	// we always have a "content" attr in plug/slots of interface
+	// type "content"
+	s, _ := attrs["content"].(string)
```

Comment:
Can you just say `return attrs["content"].(string)` here?

Reply:
I could, however it would panic if "content" is not defined in the code. This will just return an empty string. It should not be needed as we always create a content attr for type content but its slightly more defensive.

Reply:
To be pedantic, it would actually panic if it's not a string, whether it's defined or not. It's the type assertion that panics. The map will gladly return whatever it has or nil.